kmonad.cabal

```diff
@@ -123,7 +123,8 @@ library
     c-sources:
       c_src/keyio.c
     build-depends:
-      unix
+      , unix
+      , hinotify
```

Comment:
I'm surprised this list can start with a comma, but I guess if it works it works… :)